test/sql-server-cdc-old-syntax/data-utf8.td

```diff
@@ -0,0 +1,62 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+#
+# Test UTF-8 data both pre- and -post snapshot
+#
+
+$ sql-server-connect name=sql-server
+server=tcp:sql-server,1433;IntegratedSecurity=true;TrustServerCertificate=true;User ID=${arg.default-sql-server-user};Password=${arg.default-sql-server-password}
+
+$ sql-server-execute name=sql-server
+IF EXISTS (SELECT 1 FROM sys.databases WHERE name = 'test') BEGIN ALTER DATABASE test SET SINGLE_USER WITH ROLLBACK IMMEDIATE; DROP DATABASE test; END;
+CREATE DATABASE test;
```

Comment:
if we want to support full utf8 I believe we would need to update the database in sql server. per @martykulma's comment on https://github.com/MaterializeInc/database-issues/issues/9299

```suggestion
CREATE DATABASE test  COLLATE Latin1_General_100_CI_AS_SC_UTF8;
```